src/content/docs/journey-migration/migrating-logs.mdx

```diff
@@ -0,0 +1,82 @@
+---
+title: "Install log monitoring"
+metaDescription: "How to migrate your log monitoring from Datadog to New Relic."
+---
+
+import journeyLogsInstall from 'images/logs_screenshot-crop_guided-installs.png'
+
+import journeyLogsDash from 'images/logs_screenshot-crop_journey-dash.png'
+
+import journeyLogsDash2 from 'images/logs_screenshot-crop_journey-dash-two.png'
+
+import journeyLogInstall from 'images/logs_screenshot-crop_journey-install.png'
+
+Log observability is essential to both understanding and using your data to the fullest. We believe log observability should be easy to manage, easy to access, and easy to scale. New Relic lets you ingest any text-based data from any source, letting you easily manage large quantities of your data in one place. 
+
+Once you've set your logs up, you can even utilize machine learning to help you find patterns and anomalies fast without the need for manual querying. Like infrastructure and alert agents, we offer a host of guided installation options as well as preconfigured dashboards and alerts. And also like infrastructure and application agents, you can try out logging [for free](https://newrelic.com/signup).
+
+<img
+    title="log-dashboard"
+    alt="An image displaying New Relic's log monitoring dashboard"
+    src={journeyLogsDash}
+/>
+
+## Your log monitoring solution [#logs-solution]
+
+Logs in the New Relic UI are found under the **Logs** section, exactly like you're used to in Datadog. New Relic log monitoring makes getting what you need from your logs as easy as possible, helping you avoid the headache that comes from parsing massive amounts of data. 
+
+You can automatically pinpoints events, errors, traces, and spans among your connected entities with just a single click, keeping you in one place and getting you to what you're looking for that much faster. Our log monitoring solution:
+
+* Enables deep visibility into your entity data, letting you see how your system works together.
+* Helps reduce your mean time to resolution (MTTR), letting you respond to issues faster.
+* Gives you the ability to create curated charts, dashboards, and alerts, giving you as detailed a look into your data as you want.
+* Provides the ability to visualize, search, and troubleshoot performance issues in one central location without the need to switch between tools.
+
+## Get started with New Relic log monitoring [#get-started]
+
+<SideBySide>
+    <Side>
+<img
+    title="Log monitoring installations"
+    alt="A screenshot displaying some of New Relic's log monitoring guided installs"
+    src={journeyLogInstall}
+/>
+    </Side>
+    <Side>
+    Once installed, your log monitoring is done in the **Logs** section of the UI, exactly like it is in Datadog. You can use our guided installations for logs after setting up any of our [infrastructure](/docs/journey-migration/migrating-infra/) or [APM](/docs/journey-migration/migrating-apm/) agents. 
```

Comment:
I think we probably need more detail about how logs gets installed. Even just a simple bulleted list:

* Via infra
* Via APM
* Via a standalone logs integration

This install question is by far the biggest user pain point with our logs.

Ally can help with language here.